src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.server.ts

```diff
@@ -151,6 +162,30 @@ const baseQuery = gql`
             metaDescription
           }
         }
+        recentNewsCollection(where: { publicationDate_gte: $newsOldestDate }) {
+          items {
+            sys {
+              id
+            }
+            type
+            title
+            subhead
+            publicationDate
+            slug
+            byline
+          }
+        }
+        upcomingEventsCollection(where: { eventDateAndTime_gte: $eventStartDate }) {
+          items {
+            sys {
+              id
+            }
+            slug
+            shortTitle
+            eventDescription
+            eventDateAndTime
+          }
+        }
```

Comment:
These _might_ be nice to have within `$lib/fragments/`. Mainly so that we're more aware of query chunks that are identical across different pages

Reply:
Yeah good call since we're using the same query here as on Top Tier. I forget if we can include variables (e.g. `$newsOldestDate` and `$eventStartDate`) in fragments, but we should at least be able to move the queried fields into a fragment.